client/public/locales/en/translation.json

```diff
@@ -607,9 +607,9 @@
       "scope": "Scope",
       "setTargets": "Set targets",
       "source": "Source",
-      "sources": "Sources",
+      "source_plural": "Sources",
       "target": "Target",
-      "targets": "Targets",
+      "target_plural": "Targets",
```

Comment:
interesting!

Reply:
Yep, just need to include `{ count }` in the intl call to have it be able to make the choice between plural forms.